terraform/hydra-projects.tf

```diff
@@ -81,10 +81,12 @@ resource "hydra_project" "nixpkgs" {
 }
 
 resource "hydra_jobset" "nixpkgs_cuda" {
+  for_each = toset(["nixos-unstable", "master"])
```

Comment:
👎  on master, nixos-unstable small?

Reply:
do we even need nixos-unstable if nixos-unstable-small is targeted?

Reply:
I don't think so, just small should work?

Reply:
Ok, fixed in e310b5d104de3254f017ccac4718f2ac6e04fc27